addons/l10n_ec_ats/search_by_x_identificacion/__init__.py

```diff
@@ -0,0 +1,2 @@
+# -*- encoding: utf-8 -*-
+import res_partner
```

Comment:
Same as before, all your code related for the same model should be on only one file, in your models directory.